packages/@sanity/cli/src/actions/init-project/templates/nextjs/index.ts

```diff
@@ -39,27 +41,7 @@ const dataset = process.env.NEXT_PUBLIC_SANITY_DATASET
 export default defineCliConfig({ api: { projectId, dataset } })
 `
 
-export const sanityStudioPagesTemplate = `import Head from 'next/head'
```

Comment:
you are changing the name of one exported const and deleting two others. I'm assuming this is intended but just wanted to call this out to make sure.

Reply:
Yep, that's correct.
The name `sanityStudioPagesTemplate` allude to the format used to mount the studio in `pages/studio/[[...tool]].tsx` paths, which is handled by the Next.js Pages Router: https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

`next-sanity` v9 only supports App Router routes `app/studio/[[...tool]]/page.tsx` mount points: https://nextjs.org/docs/app/building-your-application/routing/defining-routes

Due to the two routes starting to diverge too much in how they handle meta data tags, and other APIs, we decided to stop supporting Pages Router and started to simplify the API surface and docs. 
As a result `sanityStudioPagesTemplate` is no longer needed and produces invalid code.

For example this export no longer exists:
```
import { metadata } from 'next-sanity/studio/metadata'
```

And since there are no longer a need to maintain two templates it no longer makes sense to specify `AppTemplate` in the name :)